notes/SDLC.md

```diff
@@ -51,7 +51,6 @@
 * [Official Scrum Guide](http://www.scrumguides.org/scrum-guide.html#team-po)
 * [Wikipedia - waterfall](https://en.wikipedia.org/wiki/Waterfall_model)
 * [Cost of a defect](http://blog.celerity.com/the-true-cost-of-a-software-bug)
-	* [support](http://istqbexamcertification.com/what-is-the-cost-of-defects-in-software-testing/)
```

Comment:
@jmgreenberg this link is pretty low quality content / ad farming (e.g. w3schools vs. mozilla for authoritative documentation about the DOM) and doesn't add much to the argument

if you can recommend a better resource, please include another one)